rpm/quota.spec

```diff
@@ -0,0 +1,224 @@
+Name:       quota
+Version:    4.05
+Release:    1
+Summary:    System administration tools for monitoring users' disk usage
+# quota_nld.c, quotaio_xfs.h:       GPLv2
+# bylabel.c copied from util-linux: GPLv2+
+# doc/quotas.ms, edquota.c:         BSD
+# COPYING:                          GPLv2 text and license declaration
+## Only in quota-warnquota binary package
+# warnquota.c:                      GPLv2+
+## Unpackaged in Sailfish
+# rquota_server.c:                  GPLv2+
+# rquota_svc.c:                     GPLv2+
+# svc_socket.c copied from glibc:   LGPLv2+
+# po/cs.po:                         GPLv2+
+## Not involved in any binary package
+# aclocal.m4:                       FSFULLR and (GPLv2+ with exception)
+# ar-lib:                           GPLv2 with exception
+# depcomp:                          GPLv2+ with exception
+# compile:                          GPLv2+ with exception
+# config.guess:                     GPLv3+ with exception
+# config.rpath:                     GPLv2+ with exception
+# config.sub:                       GPLv3+ with exception
+# configure:                        FSFUL
+# install-sh:                       MIT and Public Domain
+# m4/gettext.m4:                    GPL with exception
+# m4/iconv.m4:                      GPL with exception
+# m4/lib-ld.m4:                     GPL with exception
+# m4/lib-link.m4:                   GPL with exception
+# m4/lib-prefix.m4:                 GPL with exception
+# m4/nls.m4:                        GPL with exception
+# m4/po.m4:                         GPL with exception
+# m4/progtest.m4:                   GPL with exception
+# Makefile.in:                      FSFULLR
+# missing:                          GPLv2+ with exception
+# mkinstalldirs:                    Public Domain
+License: BSD and GPLv2 and GPLv2+
```

Comment:
I wonder if we could just simplify this to GPLv2. Also I'm not sure if it makes a lot of sense to keep that license list above (this long multiline comment) but it was inherited from Fedora's packaging.